dockers/app/Dockerfile

```diff
@@ -1,34 +1,41 @@
-FROM alpine:3.8 as production
+FROM alpine:3.11.3 as production
 MAINTAINER isard <info@isard.com>
 
 RUN apk add --no-cache \
 	yarn \
-	py3-libvirt \
-	py3-paramiko \
-	py3-lxml \
-	py3-pexpect \
-	py3-openssl \
-	py3-bcrypt \
-	py3-gevent \
-	py3-flask \
-	py3-netaddr \
-	py3-requests \
-        py3-pyldap \
-	curl \
-	openssh-client \
-	sshpass \
-	supervisor
+        py3-libvirt==5.9.0-r0 \
+	py3-paramiko==2.7.1-r0 \
+	py3-lxml==4.4.2-r0 \
+	py3-openssl==19.1.0-r0 \
+	py3-bcrypt==3.1.7-r2 \
+	py3-gevent==1.4.0-r2 \
+	py3-flask==1.0.4-r0 \
+	py3-netaddr==0.7.19-r4 \
+	py3-requests==2.22.0-r0 \
+        py3-pyldap==3.2.0-r1 \
+        libvirt-client==5.9.0-r1 \
+	curl==7.67.0-r0 \
+	openssh-client==8.1_p1-r0 \
+	sshpass==1.06-r0 \
+	supervisor==4.1.0-r0
 
 RUN apk add --no-cache --virtual .build_deps \
 	build-base \
 	python3-dev \
 	libffi-dev \
-	openssl-dev 
-RUN pip3 install --no-cache-dir pandas
-RUN apk del .build_deps
+	openssl-dev \
+        libc-dev \
+        libxml2-dev \
+        libxslt-dev \
+	gcc
 
+RUN pip3 install --upgrade pip
+RUN pip3 install --no-cache-dir pandas
 COPY dockers/app/requirements.pip3 /requirements.pip3
 RUN pip3 install --no-cache-dir -r requirements.pip3
+#RUN pip3 list --outdated --format=freeze | grep -v '^\-e' | cut -d = -f 1 | xargs -n1 pip3 install -U && exit 0
+#RUN pip3 install rethinkdb==2.3.0.post6  werkzeug==0.16.0 APScheduler==3.3.1 python-engineio==3.8.1
```

Comment:
This comments could be deleted, since they're not used for anything

Reply:
ok